src/frequenz/sdk/timeseries/logical_meter/_formula_engine.py

```diff
@@ -136,6 +145,34 @@ async def apply(self) -> Sample:
 
         return Sample(metric_ts, eval_stack[0])
 
+    async def _run(self) -> None:
+        sender = self._channel.new_sender()
+        while True:
+            try:
+                msg = await self._apply()
+            except asyncio.CancelledError:
+                logger.exception("LogicalMeter task cancelled")
+                break
+            except Exception as err:  # pylint: disable=broad-except
+                logger.warning("Formula application failed: %s", err)
```

Comment:
Please consider to write the name of the formula

Reply:
done